pkg/odo/cli/service/interface.go

```diff
@@ -0,0 +1,15 @@
+package service
+
+import "github.com/spf13/cobra"
+
+// ServiceProviderBackend is implemented by the backends supported by odo
+// It is used in "odo service create"
+type ServiceProviderBackend interface {
+	CompleteServiceCreate(options *CreateOptions, cmd *cobra.Command, args []string) error
+	ValidateServiceCreate(options *CreateOptions) error
+	RunServiceCreate(options *CreateOptions) error
+
+	CompleteServiceDelete(options *DeleteOptions, cmd *cobra.Command, args []string) error
```

Comment:
Both the implementation don't use any of the arguments passed and just return nil

Reply:
Removed from interface and implementation.